static/css/components/merge-request-table.less

```diff
@@ -112,6 +72,106 @@
   }
 }
 
+//NEW STUFF HERE
+
+.dot-wrapper {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.book-title {
+  font-weight: bold;
+  font-size: 1rem;
+}
+
+.orange-dot {
+  border-radius: 10px;
+  width: 10px;
+  background-color: @orange;
+  height: 10px;
+  margin-right: 12px;
+}
+
+.nav-bar {
+  padding: .5%;
+  background-color: @lightest-grey;
```

Comment:
```suggestion
  background-color: @grey-fafafa;
```
Figma has the correct hex values for colors.  You can then convert the hex value to HSL by using Google's color picker component:
![image](https://user-images.githubusercontent.com/28732543/229221999-6ee39b1b-78ac-41a9-a3c2-4bbd8f366406.png)